src/main/java/core/basesyntax/servise/ShopServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.servise;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.servise.operation.OperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ShopServiceImpl implements ShopService {
+    private final OperationStrategy operationStrategy;
+    private final Map<String, Integer> storage = new HashMap<>(); // Хранилище фруктов
```

Comment:
According to the encapsulation principle, the storage should not be exposed. Consider changing the visibility of the storage field to private and provide a way to access the data safely, if needed.